flow/src/org/labkey/flow/webparts/FlowSummary.jsp

```diff
@@ -115,29 +115,31 @@
     }
 </style>
 
-<script type="text/javascript">
+<script type="text/javascript" nonce="<%=getScriptNonce()%>">
     function createRenderer(detailsURL, countProperty, countLabel)
     {
+        const h = LABKEY.Utils.encodeHtml;
+
         return function (el, json) {
             var html = "<table border='0'>";
             for (var i = 0; i < <%=DISPLAY_MAX_ROWS%> && i < json.rows.length; i++) {
                 var row = json.rows[i];
                 var name = row.Name.value;
                 var url = row.Name.url;
-                var comment = row["Flag/Comment"].value ? " title='" + row["Flag/Comment"].value + "'" : "";
+                var comment = row["Flag/Comment"].value ? " title='" + h(row["Flag/Comment"].value) + "'" : "";
                 var iconCls = row["Flag/Comment"].value ? <%=q(FlagColumnRenderer.flagEnabledCls())%> : "";
 
                 html += "<tr>" +
-                        "<td>" + (iconCls ? "<a" + comment + " href='" + url + "'><i class='" + iconCls + "'/></a>" : "") + "</td>" +
-                        "<td nowrap><a" + comment + " href='" + url + "'>" + name + "</a><td>";
+                        "<td>" + (iconCls ? "<a" + comment + " href='" + h(url) + "'><i class='" + h(iconCls) + "'/></a>" : "") + "</td>" +

```

Comment:
Shouldn't this get q() too/instead?

Reply:
It would be good to write a unit test that ensures the correct behavior or htmlEncode() though.

Reply:
update: I checked that encodeURI() handles backslash, might need to add that to htmlEncode.

Reply:
Ah, OK. I lost track of which h() this was calling.